tensorflow/g3doc/tutorials/mnist/pros/index.md

```diff
@@ -409,8 +410,14 @@ for i in range(20000):
     print("step %d, training accuracy %g"%(i, train_accuracy))
   train_step.run(feed_dict={x: batch[0], y_: batch[1], keep_prob: 0.5})
 
-print("test accuracy %g"%accuracy.eval(feed_dict={
-    x: mnist.test.images, y_: mnist.test.labels, keep_prob: 1.0}))
+acc = 0 
+num_batch = mnist.test.num_examples/50
+for i in range(num_batch):
+  batch = mnist.test.next_batch(50)
+  acc += accuracy.eval(feed_dict={x: batch[0], y_: batch[1], keep_prob: 1.0})
+
+acc /= num_batch
+print("test accuracy %g"%acc)
```

Comment:
@rmekdma you are splitting the mnist test set into 50 batches and testing accuracy of each batch individually.  Then you average the each of those batch accuracy values together into an overall "test accuracy".  But:

- If all batches are of identical size, then this will be exactly equivalent to the existing unbatched accuracy calculation, just slower.
- And if the batches are not all identically sized, the overall test accuracy value will be incorrect.

What is the improvement you are trying to make?